core/types/block.go

```diff
@@ -28,12 +28,14 @@ import (
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/rlp"
 )
 
 var (
 	EmptyRootHash  = common.HexToHash("56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421")
 	EmptyUncleHash = rlpHash([]*Header(nil))
+	EmptyCodeHash  = crypto.Keccak256Hash(nil)
```

Comment:
```suggestion
	EmptyCodeHash  = crypto.Keccak256Hash(nil) // c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470
```
Also, is there any difference between `EmptyUncleHash` and `EmptyRootHash`?

Reply:
I prefer having explicitly EmptyX EmptyY, EmptyZ hash even if they are the same, otherwise it gets hard to see if something's a bug or just the same number.

Reply:
Added

Reply:
`EmptyUncleHash` and `EmptyRootHash` is different